tools/cli-bootstrap/dockerfiles/debian8/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:8.0
```

Comment:
I was just suggesting that we add them with the follow-up PR instead, since a lot of these don't actually work in their current state (they all use `apt-get` commands). It's not a big deal either way.